docs/source/getstarted/installation.md

```diff
@@ -62,5 +62,6 @@ pip install -e .
 ```
 
 ## Optional Dependencies
-Olive has optional dependencies that can be installed to enable additional features. Please refer to [extra dependencies](https://github.com/microsoft/Olive/blob/main/olive/extra_dependencies.json)
-for the list of extras and their dependencies.
+Olive has optional dependencies that can be installed to enable additional features. Please refer to
+[Olive package config](https://github.com/microsoft/Olive/blob/main/olive/olive_config.json) for the list of extras
```

Comment:
https://github.com/microsoft/Olive/pull/1028#discussion_r1533084725
Similar to this. You may need a separated PR to update the link. Otherwise, the docs test CI will fail.

Reply:
Not if #1028 goes in ahead of this. #1028 introduces the olive_config.json and this one deprecates extra_dependencies.json.